docs/connecting-your-data/assignment-tables/randomization-sdk.md

```diff
@@ -0,0 +1,163 @@
+---
+sidebar_position: 1
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Create assignment table with Eppo Randomization SDK
```

Comment:
Could we change the title to just "Eppo Randomization SDK" and also change the subheading like described here:

https://eppo-group.slack.com/archives/C035Z4W1LJZ/p1654580234143149